modules/oadp-kopia-algorithms-benchmarking.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/overriding-kopia-algorithms.adoc          
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-kopia-algorithms-benchmarking_{context}"]
+= Benchmarking Kopia hashing, encryption, and splitter algorithms
+
+You can run Kopia commands to benchmark the hashing, encryption, and splitter algorithms. Based on the benchmarking results, it is possible to select the most suitable algorithm for your workload.
```

Comment:
Should we mention that the benchmark needs to be ran from the cluster itself?

Reply:
@weshayutin what is your take on it? Should we mention must-gather and the `kopia` CLI in there, so similarly to what was used in one of the tests:

```yaml
apiVersion: v1
kind: Pod
metadata:
  name: oadp-must-gather-pod
  labels:
    purpose: kopia-benchmark
spec:
  containers:
  - name: oadp-must-gather-container
    image: registry.redhat.io/oadp/oadp-mustgather-rhel8:v1.1
    command: ["sleep"]
    args: ["infinity"] 
```

Reply:
@mpryc hold up.. you are saying there is a difference when one run benchmark from laptop vs. a pod?  This is news.   Help me understand that bit please.

Reply:
**YES** The results  can vary significantly depending on the machine. Things like CPU speed, available RAM, disk speed, current I/O load and so on.

Here is an upstream comment from which hopefully makes things clear:
   https://kopia.discourse.group/t/questions-i-cannot-find-an-answer-to/428/6

With the above I think that benchmarks may and will vary depending on the resources, however the % increase or decrease between algorithms should be similar.

Reply:
@weshayutin @mpryc - I will add this information. Something like a note saying -  results differ from pod vs laptop and if you want to benchmark from a pod here is how you configure a pod - add the 'must-gather' config snippet . Thanks.

Reply:
Done.